plugin/pkg/admission/storageclass/default/admission.go

```diff
@@ -88,7 +88,7 @@ func (a *claimDefaulterPlugin) SetInternalClientSet(client internalclientset.Int
 	}
 }
 
-// Validate ensures an authorizer is set.
+// Validate ensures clientset and store are set.
```

Comment:
my issue is that the comment is not actually correct as updated.  it validates 3 things, not 2.